packages/debugger/src/tokens.ts

```diff
@@ -87,6 +87,18 @@ export interface IDebugger {
     variablesReference: number
   ): Promise<DebugProtocol.Variable[]>;
 
+  /**
+   * Request rich representation of a variable.
+   *
+   * @param variableName The variable name to request
+   * @param variablesReference The variable reference to request
+   * @returns The mime renderer data model
+   */
+  inspectRichVariable(
```

Comment:
The debug message is called `richInspectVariables`, should the public method on the service named the same (in singular form)?

Not strong opinion, just raising the question before this gets in the pubic `IDebugger` API.